server/graphql/v1/mutations.js

```diff
@@ -717,6 +720,44 @@ const mutations = {
       return paymentMethodsMutation.removePaymentMethod(args.id, req.remoteUser);
     },
   },
+  editNotifications: {
+    type: new GraphQLList(NotificationType),
+    args: {
+      id: { type: new GraphQLNonNull(GraphQLInt) },
+      notifications: { type: new GraphQLList(NotificationInputType) },
+    },
+    resolve(_, args, req) {
+      return editNotifications(args, req.remoteUser);
+    },
+  },
+  createNotification: {
+    type: NotificationType,
+    args: {
+      notification: NotificationInputType,
+    },
+    resolve(_, args, req) {
+      return createNotification(args, req.remoteUser);
+    },
+  },
+  createWebhook: {
+    type: NotificationType,
+    args: {
+      collectiveSlug: { type: new GraphQLNonNull(GraphQLString) },
+      notification: NotificationInputType,
```

Comment:
This doesn't work, we need to be explicit here:

```suggestion
      notification: { type: NotificationInputType },
```